api/webhook/github.ts

```diff
@@ -215,23 +215,27 @@ async function pushEvent(
   const ref = rawRef.replace(/^refs\/tags\//, "");
   const description = webhook.repository.description ?? "";
   const starCount = webhook.repository.stargazers_count;
-  const versionPrefix = decodeURIComponent(
-    event.queryStringParameters?.version_prefix ?? "",
+  const versionPrefixFilter = decodeURIComponent(
+    event.queryStringParameters?.version_prefix_filter ??
+      event.queryStringParameters?.version_prefix ?? "",
```

Comment:
took me a moment to realize that the `version_prefix` parameter is effectively being renamed to `version_prefix_filter`. I think we can move add a simple if statement at the beginning of the handler to check for this case, like

```typescript
if (event.queryStringParameters?.version_prefix && !event.queryStringParameters?.version_prefix_filter) {
  event.queryStringParameters.version_prefix_filter = event.queryStringParameters.version_prefix;
}
```